vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -225,8 +228,89 @@
                 $(".open").css('overflow-y', 'scroll');
                 $(".open").css('height', '400px');
             }
-            
+
+            function biblioBlockDoubleClick(e){
+               $(".EasyMDEContainer").remove();
+         	   $(e.target).closest('.biblioDivStaff').addClass("open");
+         	   var description = $("div.open p:eq(0)").text();
+           	   defaultValue = description;
+           	
+           	   var title = $("div.open p span:eq(0)").text();
+           	   var desc = $("div.open p span:eq(1)").text();  
+	      	    //showing the edit bibliography form with all the respective values
+      	       $("#editBiblioAlert").show();
+      	
+      	       $("#editTitleBiblio").val(title);
+      	       $("#editDescBiblio").val(desc);
+           		
+           	    formDataCancel = new FormData();
+           	    formDataCancel.append('biblioTitle', title);
+           	    formDataCancel.append('description', desc);
+            	    //remove card border
+                $(".open").css('display', 'none');
+                $(".open").css('overflow-y', 'scroll');
+                $(".open").css('max-height', '200px');	
+             } 
+             
+             function referenceBlockDoubleClick(e){
+            	 	currentRefId = $(this).find('#deleteReferenceId').val();
+            	    $(".EasyMDEContainer").remove();
+	         	    $(e.target).closest('.referenceDivStaff').addClass("open");
+	         	    var description = $("div.open p:eq(0)").text();
+             	    defaultValue = description;
+             	
+             	    var title = $("div.open p span:eq(0)").text();
+                    var author = $("div.open p span:eq(1)").text();
+                    var year = $("div.open p span:eq(2)").text();
+                    var journal = $("div.open p span:eq(3)").text();
+                    var url = $("div.open p span:eq(4)").text();
+                    var volume = $("div.open p span:eq(5)").text();
+                    var issue = $("div.open p span:eq(6)").text();
+                    var pages = $("div.open p span:eq(7)").text();
+                    var editors = $("div.open p span:eq(8)").text();
+                    var type = $("div.open p span:eq(9)").text();
+                    var note = $("div.open p span:eq(10)").text(); 
+             	
+             	    //showing the edit bibliography form with all the respective values
+		    	    $("#editReferenceAlert").show();
+		    	    var alert = $('<input type="hidden" id="currentBiblioId">');
+		    		/* $(alert).attr( 'value', $(this).parent().parent().parent().attr("id"));*/
+		    		$(alert).attr( 'value', $(e.target).closest('.biblioDivStaff').attr("id"));
```

Comment:
this seems unnecessarily complicated and brittle. 
a) why is the variable called "alert"? It does not contain an alert, does it?
b) if I understand this right, all that this code is trying to do is that the id of the reference that should be deleted in the delete form, right? If so, the block that was clicked on should just have a data attribute that holds the id of the reference that should be deleted. Then get that value here and set it in the input. 
c) The input element should probably be already in the modal, and then just set the value here. No need to create a new input element every time.

Reply:
The same is true for all the other cases where there is similar code here in the file.